src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -305,7 +305,11 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
         const unsubscribeSearchShortcut = KeyboardShortcut.subscribe(
             searchShortcutConfig.shortcutKey,
             () => {
-                Modal.close(Session.checkIfActionIsAllowed(() => Navigation.navigate(ROUTES.CHAT_FINDER)));
+                Modal.close(
+                    Session.checkIfActionIsAllowed(() => Navigation.navigate(ROUTES.CHAT_FINDER)),
+                    true,
```

Comment:
what was isNavigating meant for?

Reply:
`isNavigating` is later passed to closeModals function, however it's `true` as a default so, it doesn't change anything here